starter-code/app.js

```diff
@@ -1,5 +1,60 @@
+'use strict';
+
 const express = require('express');
 const app = express();
-const Chuck  = require('chucknorris-io');
+const Chuck = require('chucknorris-io');
 const client = new Chuck();
+const expressLayouts = require('express-ejs-layouts');
+
+// app.get('/', (req, res, next) => {
+//   res.send('Hello');
+// });
+
+
+// -- setup
+app.use(express.static('public'));
+app.use(expressLayouts);
+app.set('layout', 'layouts/main-layout');
+app.set('views', __dirname + '/views');
+app.set('view engine', 'ejs');
+
+
+// -- routes
+
+app.get('/random', (req, res) => {
+  client.getRandomJoke()
+    .then((response) => {
+      const data = {
+        joke: response
+      }
+      res.render("random", {
```

Comment:
You don't need to wrap the "data" variable in curly braces because it is already an object.